cpp/velox/substrait/SubstraitToVeloxExpr.cc

```diff
@@ -85,6 +85,22 @@ ArrayVectorPtr makeArrayVector(const VectorPtr& elements) {
   return std::make_shared<ArrayVector>(elements->pool(), ARRAY(elements->type()), nullptr, 1, offsets, sizes, elements);
 }
 
+MapVectorPtr makeMapVector(const VectorPtr& keyVector, const VectorPtr& valueVector) {
+  BufferPtr offsets = allocateOffsets(1, keyVector->pool());
+  BufferPtr sizes = allocateOffsets(1, keyVector->pool());
+  sizes->asMutable<vector_size_t>()[0] = keyVector->size();
+
+  return std::make_shared<MapVector>(
+      keyVector->pool(),
```

Comment:
this will allocate memory from keyVector's pool, the pool from keyVec or valueVec should be always the same?

Reply:
yes, they are always same